README.md

```diff
@@ -142,10 +145,11 @@ teleop:                        # Parameters for the Teleoperate mode.
   default_increment: 0.1       # Default velocity increment when pressing a key.
   increment_step: 0.1          # Step for increasing the velocity increment.
   cmd_vel_topic: cmd_vel       # Topic on which to publish the velocity commands.
+  force_publish_on_idel: false # If false keep publishing 0 velocities, only publish once otherwise
```

Comment:
idle typo

Reply:
the name of the parameter is cryptic.
I suggest keep_publishing_cmd_vel_when_idle

Reply:
publish_cmd_vel_when_idle, or publish_when_idle as its already in the cmd_vel namespace?